### PR TITLE
Feature/show random cover button

### PR DESCRIPTION
**Pull request by coopterrones**
## What is the change?
adding iteration 1 and iteration 2
 ## What does it fix?
Broken embrace.jpg and overlay position bug fixed 
## Is this a fix or a feature? 
feature with minor bug fixes
## Where should the reviewer start? 
main.js beginning
## How should this be tested?
Open index.html and run through button clicks and navigation.

### CODE REVIEW

**Review by mdflynn — APPROVED** [review]
This looks great! Great naming conventions!

File: src/main.js [hunk 1]
```diff
@@ -3,6 +3,14 @@ var coverImage = document.querySelector('.cover-image');
 var coverTitle = document.querySelector('.cover-title');
 var discriptorTagLine1 = document.querySelector('.tagline-1');
 var discriptorTagLine2 = document.querySelector('.tagline-2');
+var showNewCoverButton = document.querySelector('.random-cover-button');
+var makeOwnCoverButton = document.querySelector('.make-new-button');
+var mainPageDisplay = document.querySelector('.main-cover');
+var displayUserForm = document.querySelector('.form-view');
+var saveCoverButton = document.querySelector('.save-cover-button');
```
Comment (Ish451) [inline src/main.js:10]: Everything looks nice and cleaned up now. This was a great learning experience for us when it comes to naming conventions.

File: src/main.js [hunk 2]
```diff
@@ -22,12 +35,38 @@ function displayMainPageCover() {
...
+  mainPageDisplay.classList.add('hidden');
+  viewSavedButton.classList.add('hidden');
+  saveCoverButton.classList.add('hidden');
+  homeButton.classList.remove('hidden');
+  showNewCoverButton.classList.add('hidden');
+  displayUserForm.classList.add('hidden');
+}
+
+function homeButtonFunction() {
+  mainPageDisplay.classList.remove('hidden');
```
Comment (Ish451) [inline src/main.js:64]: Everything looks with the removing and adding.

File: src/main.js (hunk 1 above)
Comment (Ish451) [inline src/main.js:10]: The variables look like they're all correlating correctly!

**Review by Ish451 — APPROVED** [review]
Everything looks nice and cleaned up. This was a good learning experience for us with the naming conventions.

**Pull request merged**